### DISCUSSION TITLE
Is it possible to set background colour of a DataGrid row based on its content?

### DISCUSSION THREAD

**Discussion by sturlath** [Megabit/Blazorise#3426] [Q&A] [answered]
I have a grid with events that should start at a certain time and I would like to be able to colour the row if that time has passed as a warning reminder for them to act.

I could at least not find any obvious way to do this.

**Comment by David-Moreira** [comment, marked as answer]
hmm in theory you can use the new .NET6 timer and `RowStyling="@OnRowStyling"` making sure to call `dataGridRef.Refresh` or `dataGridRef.Reload`.

Could you try it like that?

**Reply by stsrki** [reply]
Also, can try the `CellStyling`.

**Reply by sturlath** [reply]
hmm ok thanks I´ll try out these suggestions today and share the results. Thank you

**Reply by sturlath** [reply]
Ok this looks super simple! I was just looking for properties in the wrong places and I couldn´t find [any "row styling" in the docs](https://preview.blazorise.com/docs/extensions/datagrid/getting-started) but now I found it [here](https://github.com/Megabit/Blazorise/blob/0433d2ac02c4d89ea3bab0ba29f153649de89dcf/Demos/Blazorise.Demo/Pages/Tests/DataGrid/SelectionStylingPage.razor#L84).
